roboteam_ai/src/skills/Pass.cpp

```diff
@@ -32,10 +32,10 @@ Pass::Status Pass::onUpdate() {
     bool closeToBall = (robot->pos - ball->pos).length() < CLOSE_ENOUGH_TO_BALL;
 
     if(!closeToBall && !passInitialized) {
-        auto pva = robot->getNumtreePosControl()->getPosVelAngle(robot, ball->pos);
-        command.x_vel = pva.vel.x;
-        command.y_vel = pva.vel.y;
-        command.w = pva.angle;
+        auto robotCommand = robot->getNumtreePosControl()->getPosVelAngle(robot, ball->pos);
```

Comment:
I'd rather see explicit typing where possible.

Reply:
unless the type name is unnecessarily long which it really is not here.

Reply:
Auto is fine

Reply:
Yes it is not a problem, but why change the type from a known type to auto? I think it's fine if it is hard to read or a long declaration or so but should not be necessary here as it is just PosVelAngle

Reply:
I think it's more readable if you can explicitly see the type that is returned in this case.